spaces/domains.json

```
@@ -194,4 +194,5 @@
   "vote.lattice.exchange": "latticegov.eth",
   "gov.libertas.network": "libertas.eth",
   "voting.pantherswap.com": "pantherswap.eth"
+  "blxpr.primedao.io": "blxpr.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "voting.pantherswap.com": "pantherswap.eth",
  "blxpr.primedao.io": "blxpr.eth"
```